sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/OnSdlChoiceChosen.java

```diff
@@ -85,42 +85,12 @@ public String toString() {
 		}
 	}
 	
-	public class SdlChoice {

```

Comment:
We can't just remove this as it is a public class. Which would require a major version change of the library.